Prettier User Choice on index page

LucianPetri opened the pull request:
This PR is linked to this issue: Better UI Choice #74

LucianPetri commented:
Finished the Component, in case it looks better without the grayscale text just have undo last commit. IMO looks cooler now.

fozziethebeat commented on website/src/components/UserChoice.tsx:
```diff
@@ -0,0 +1,51 @@
+import { Card, CardBody, Flex, Heading } from "@chakra-ui/react";
```
The  name `Choice` feels a little odd.  Is there a more intuitive name? Maybe something like `TaskOptions` as the component and then `TaskOption` for each individual thing that can be chosen.  It makes more sense to me at least to tie it to tasks rather than the user.

fozziethebeat commented on website/src/components/UserChoice.tsx:
```diff
@@ -0,0 +1,51 @@
+
+const UserChoice = () => {
+  return (
+    <Flex gap={10} wrap="wrap" justifyContent="center">
```
A plan I have is to have two sections: `create` and `evaluate` type tasks.  What about restructuring (and likely `Option` its own component) this a bit so it's a higher level component where someone can do the following:

```
<TaskOptions title="Create">
  <TaskOption title="Summarize Stories" link="..." icon={....} />
  ...
</TaskOptions>

<TaskOptions title="Evaluate">
  <TaskOption title="Rate Summaries" link="..." icon={....} />
  ...
</TaskOptions>
```

LucianPetri replied:
Great idea! i shall rename the components and restructure them. This was a 4AM idea so my naming convention wont be on point, I'll have them done tomorrow. Also I wanted to suggest to maybe open more tasks for UI improvements? I would like to make more things pretty and also add a dark mode.

fozziethebeat replied:
Sure, I can do a sweep through the current state of things and start listing more opportunities.

fozziethebeat reviewed (changes requested):
Thank you for helping clean up the logged in landing page! I like the use of chakra.

I have some suggestions on how to make this a bit more flexible for the growing set of tasks that'll be added.

LucianPetri commented:
Ok, I added a TaskSelection, TaskOptions and TaskOption Component. Didn't want clutter the index so I threw everything in the TaskSelection Logic. Also merged the new paths and added them.

LucianPetri commented:
On the way, I also found that the Footer overlaps when on mobile screen while having multiple tasks. I Can do this after.

LucianPetri commented:
Also if There's to be added more stuff on index page, I have in mind another style to view the task selection to give more room for other stuff to be added. Hit me up with more details if required.

LucianPetri reviewed (commented):
Refactor UserChoice -> TaskSelection
